### PR TITLE
improvement: address Bootstrap conflict by renaming our column class (via #4574)

### LINKED ISSUE #4574: How to cope with bootstrap4 as surrounding framework for integration?
How do allow embedding Swagger-UI, default theme (which I think is just fine) into a webapp which uses bootstrap4?  Looks like classes conflict and the end result is that the Swagger-UI renders very very strangely.  Its mostly okay but things like its endpoint detail layouts is totally messed up.

I'm really happy with the default theme for my use-case, but the webapp its inside of is bootstrap4 and its appears that things like "col" conflict and end up having the swagger display rendering very strangely.

monosnap 2018-05-20 15-56-22

Is there a simple solution to this?

### LINKED ISSUE #5498: Reponse details to not take column width
### Q&A (please complete the following information)
 - OS: Windows 10
 - Browser: Chrome
 - Version: 75.0.3770.142
 - Method of installation: npm
 - Swagger-UI version: 3.22.3
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration

Example Swagger/OpenAPI definition:
```yaml
openapi: "3.0.0"
info:
  version: 1.0.0
  title: Swagger Petstore
  license:
    name: MIT
servers:
  - url: http://petstore.swagger.io/v1
paths:
  /pets:
    get:
      summary: List all pets
      operationId: listPets
      tags:
        - pets
      parameters:
        - name: limit
          in: query
          description: How many items to return at one time (max 100)
          required: false
          schema:
            type: integer
            format: int32
      responses:
        '200':
          description: A paged array of pets
          headers:
            x-next:
              description: A link to the next page of responses
              schema:
                type: string
          content:
            application/json:    
              schema:
                $ref: "#/components/schemas/Pets"
        default:
          description: unexpected error
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Error"
    post:
      summary: Create a pet
      operationId: createPets
      tags:
        - pets
      responses:
        '201':
          description: Null response
        default:
          description: unexpected error
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Error"
  /pets/{petId}:
    get:
      summary: Info for a specific pet
      operationId: showPetById
      tags:
        - pets
      parameters:
        - name: petId
          in: path
          required: true
          description: The id of the pet to retrieve
          schema:
            type: string
      responses:
        '200':
          description: Expected response to a valid request
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Pet"
        default:
          description: unexpected error
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Error"
components:
  schemas:
    Pet:
      required:
        - id
        - name
      properties:
        id:
          type: integer
          format: int64
        name:
          type: string
        tag:
          type: string
    Pets:
      type: array
      items:
        $ref: "#/components/schemas/Pet"
    Error:
      required:
        - code
        - message
      properties:
        code:
          type: integer
          format: int32
        message:
          type: string
```

Swagger-UI configuration options:
```js
import React from "react";
import PropTypes from "prop-types";
import SwaggerUI from "swagger-ui-react";
import "swagger-ui-react/swagger-ui.css";

import apis from "../config/apis";

const Api = ({ match }) => {
  const api = apis.find(a => a.id === match.params.apiId);
  return <SwaggerUI url={api && api.oasUrl} />;
};

Api.propTypes = {
  match: PropTypes.shape({
    params: PropTypes.shape({
      apiId: PropTypes.string.isRequired
    }).isRequired
  }).isRequired
};
```
export default Api;
```jsx
import React from "react";
import { Navbar, Nav, NavDropdown } from "react-bootstrap";
import { BrowserRouter, Route } from "react-router-dom";
import Api from "./pages/Api";
import "./App.css";

import apis from "./config/apis";

export default () => (
  <BrowserRouter basename="/api-documentation-app">
    <Navbar bg="pp-dark" expand="lg" variant="dark">
      <Navbar.Brand href="/api-documentation-app">
        API Documentation
      </Navbar.Brand>
      <Navbar.Toggle aria-controls="basic-navbar-nav" />
      <Navbar.Collapse id="basic-navbar-nav">
        <Nav className="mr-auto">
          <NavDropdown title="API" id="basic-nav-dropdown">
            {apis.map(api => (
              <NavDropdown.Item
                key={api.id}
                href={`/api-documentation-app/api/${api.id}`}
              >
                {api.name}
              </NavDropdown.Item>
            ))}
          </NavDropdown>
        </Nav>
      </Navbar.Collapse>
    </Navbar>
    <Route path="/api/:apiId" exact component={Api} />
  </BrowserRouter>
);
```

### Describe the bug you're encountering
When I open API and try a method the response is displayed in very narrow column:
image

### To reproduce...

1. Go to API
2. Choose any method
3. Click try it out
4. Click execute
5. Error: image

### Expected behavior
Response details should take all the column width space

### Screenshots
image

### Additional context or thoughts
I tweaked the CSS when live in Chrome and it worked.

### PR DESCRIPTION
**Pull request by thierrydallacroce**
Both Bootstrap and SwaggerUI rely on a `col` class. When SwaggerUI is added to a site using Bootstrap, the width of some columns is negatively impacted, as seen in screenshots in bugs [4574](https://github.com/swagger-api/swagger-ui/issues/4574) and [5498](https://github.com/swagger-api/swagger-ui/issues/5498).

### Description

As suggested by @shockey  in their comment [here](https://github.com/swagger-api/swagger-ui/issues/4574#issuecomment-480405337), I replaced the `col` css class to `swui-col`.

### Motivation and Context

Fixes #4574
Fixes #5498

### How Has This Been Tested?

I ran `npm run test` after my first round of changes, then modified one test which broke since it used `.col` to traverse the DOM, then ran the full test suite again.

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by shockey** [comment]
Actually, after taking a closer look, I don't think we're even using `.col` in our stylesheets. We may be able to remove this entirely...

**Comment by thierrydallacroce** [comment]
> Actually, after taking a closer look, I don't think we're even using `.col` in our stylesheets. We may be able to remove this entirely...

No problem, I will remove it then, and update this PR once done.

**Comment by shockey** [comment]
@thierrydallacroce sounds good! for the Cypress tests, I think you can just sub out `.swui-col` with `.parameters-col_description`. Ping me if it gives you any trouble, we could look at using a `data-` attribute to target the element instead if Cypress needs a hint to grab what it needs.

thanks!

**Comment by thierrydallacroce** [comment]
I removed the column class entirely and used a css attribute selector in Cypress' get() to target either `parameters-col_description` or `parameters-col_name`.

Passed entire test suite twice -- the OAuth2 test timed out once.
Squashed and applied on top of latest upstream/master.

Let me know if that's okay

File: test/e2e-cypress/tests/bugs/5452.js [hunk 1]
```diff
@@ -8,12 +8,12 @@ describe("#5452: <Select /> crashing in Parameters", function() {
       cy.visit("http://localhost:3230/?url=/documents/bugs/5452/openapi.yaml")
         .get("#operations-default-get_endpoint")
         .click()
-        .get(".parameters > tbody > tr > td[class^='parameters-col_'] > select")
+        .get(".parameters > tbody > tr > .parameters-col_description > select")
```
Comment (shockey) [inline test/e2e-cypress/tests/bugs/5452.js]: I think this will suffice, let's see if CI passes 😄

**Comment by thierrydallacroce** [comment]
Apologies, I tried replying to your comment but hit Resolve Conversation instead.

You're right, it did suffice, and passed testing :)

**Comment by shockey** [comment]
thanks @thierrydallacroce!

**Comment by Mykolas-Zenitech** [comment]
Good to see this merged. Which version should I expect the fix to be released in?

**Comment by webron** [comment]
@Mykolas-Zenitech we push a release every 1-2 weeks. Since there wasn't one last Friday, it's very likely there will be one this Friday.

**Pull request merged**